test/override.js

```diff
@@ -33,6 +33,24 @@ describe('@override@', function () {
       done();
     });
   });
+  it("@builders@ overrides tgtBrowser abstraction", function (done) {
```

Comment:
I can try that. The problem is that, in CI, there is no valid webdriver server URL I can use, so the setup will fail. The one test does verify the logic pattern, if not both implementations of it.